index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title> Leslie Portfolio project</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body class="body">
+    <header class="header">
+      <nav class="navigate">
+        <a href=""><img src="images/John Doe.png" alt="" /></a>
+        <a href=""><img src="images/Union.png" alt="" /></a>
+      </nav>
+    </header>
+    <div class="heading-section">
+      <div class="heading1">
+        <h1 class="heading">Hey There <br />I’m Leslie</h1>
+      </div>
+      <div class="describe">
+        <h2 class="describe">I am a Software Developer</h2>
+      </div>
+      <p class="headline">
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
+      <div class="social">
+        <a href="https://mobile.twitter.com/Leslie"
+          ><img class="social-icons" src="images/twitter.png" alt=""
+        /></a>
+        <a href="https://www.linkedin.com/in/LeslieAine"
+          ><img class="social-icons" src="images/linkedin.png" alt=""
+        /></a>
+        <a href=""><img class="social-icons" src="images/medium.png" alt=""
+        /></a>
+        <a href="https://github.com/LeslieAine"
+          ><img class="social-icons" src="images/github.png" alt=""
+        /></a>
+        <a href=""><img class="social-icons" src="images/angelist.png" alt=""
+        /></a>
+      </div>
```

Comment:
- [ ] I would kindly suggest you make use of a `<ul>` tag as a container to group all your icons and each icon should have an `<li>` tag with an anchor tag i.e `<a>`.  Apart from being semantically correct, the `<a>` tag will let you connect your social networks with the icons, using href.

Example: 👇 
```HTML
<ul class="social">
    <li>
        <a href="https://mobile.twitter.com/Leslie"><img src="images/twitter.png" alt=""/></a> 
    </li>
    <li>
        <a href="https://www.linkedin.com/in/LeslieAine"><img src="images/linkedin.png" alt=""/></a>
     </li>
</ul>
```